.github/release.yml

```diff
@@ -0,0 +1,32 @@
+changelog:
```

Comment:
when will `release.yml` be invoked?

Reply:
on github release page, there is a button for triggering it manually.